config/install/docker.ci.api.dynamic-store-off.yml

```diff
@@ -69,23 +75,13 @@ sections:
         propel-migration-cleanup:
             command: 'vendor/bin/console propel:migration:delete'
 
-    demodata:
-        queue-setup:
-            command: 'vendor/bin/console queue:setup'
-
         init-database:
             command: 'vendor/bin/console setup:init-db'
 
+    demodata:
         import-demodata:
             command: 'vendor/bin/console data:import --config=data/import/common/minimal.yml'
 
-        setup-search-create-sources:
-            command: 'vendor/bin/console search:setup:sources'
-
-        publish-and-sync:
-            command: 'vendor/bin/console queue:worker:start --stop-when-empty'
-            timeout: 2000
```

Comment:
Don't we longer need it?

Reply:
Here is the old deployment yaml https://github.com/spryker-shop/suite/blob/master/config/install/docker.ci.api.yml and we do not have it there.

Github does not show the difference correctly. It was not previously `docker.ci.api.dynamic-store.yml` but `docker.ci.api.yml` which stood for DS off.